src/CompatibilityExtensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿#region Apache License
+//
+// Licensed to the Apache Software Foundation (ASF) under one or more 
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership. 
+// The ASF licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with 
+// the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+#if NETSTANDARD1_3
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Reflection;
+using System.Threading;
+using System.Xml;
+
+namespace log4net
+{
+    internal static class CompatibilityExtensions
```

Comment:
It's in the root namespace to avoid sprinkling something like this throughout the project:
```cs
#if NETSTANDARD1_3
using log4net.Extensions
#endif
```
My thinking here is that these extensions implement simple method substitutions, aren't part of log4net's public surface, and are only visible to a developer working specifically in the `netstandard1.3` context (where they're unlikely to be noticed), so it's convenient to keep them together.